packages/snet_cli/snet/snet_cli/mpe_service_metadata.py

```diff
@@ -40,66 +42,117 @@
 import secrets
 
 from collections import defaultdict
+from enum import Enum
 
 from snet.snet_cli.utils import is_valid_endpoint
 
 
+# Supported Asset types
+class AssetType(Enum):
+    HERO_IMAGE = "hero_image"
+    IMAGES = "images"
+    DOCUMENTATION = "documentation"
+    TERMS_OF_USE = "terms_of_use"
+
+    @staticmethod
+    def is_asset_having_single_value(asset_type):
```

Comment:
Suggestion: `is_asset_having_single_value()` -> `is_single_value()`
```
def is_single_value(asset_type):
    if asset_type in [a.value for a in AssetType] and asset_type != AssetType.IMAGES.value:
        return True
```

Reply:
Personally I prefer more self-explanatory names. So personally I would prefer ```mpe_service_metadata::is_asset_having_single_value``` rather ```mpe_service_metadata::is_single_value```.

Reply:
What do you mean by `mpe_service_metadata::is_asset_having_single_value`?
I think the class name + method is self explanatory:
`AssetType.is_single_value()`.
But this is minor (in fact it was my attempt to put this huge if line into < 80 chars)

Reply:
Sorry. My bad. You are right.

Reply:
agree. class name + method should be  self explanatory.